apps/formations-initiales/src/chargement/domain/service/charger-formations-initiales.domain-service.ts

```diff
@@ -0,0 +1,75 @@
+import { UnJeuneUneSolution } from "@formations-initiales/src/chargement/domain/model/1jeune1solution";
+import { FluxChargement } from "@formations-initiales/src/chargement/domain/model/flux";
+import {
+	FormationsInitialesChargementRepository,
+} from "@formations-initiales/src/chargement/domain/service/formations-initiales-chargement.repository";
+
+import { DateService } from "@shared/src/domain/service/date.service";
+
+export class ChargerFormationsInitialesDomainService {
+	static readonly INDENTATION_JSON: number = 2;
+	static readonly REMPLACANT_JSON: null = null;
+
+	constructor(private readonly formationsInitialesRepository: FormationsInitialesChargementRepository, private readonly dateService: DateService) {
+	}
+
+	async charger(flux: FluxChargement): Promise<void> {
+		const formationsInitialesASauvegarder = await this.formationsInitialesRepository.recupererFormationsInitialesASauvegarder(flux.nom);
+		const formationsInitialesASupprimer = await this.formationsInitialesRepository.recupererFormationsInitialesASupprimer(flux.nom);
+		const formationInitialesASupprimerEnErreur = await this.formationsInitialesRepository.supprimer(formationsInitialesASupprimer, flux.nom);
+		const formationsInitialesASauvegarderEnErreur = await this.formationsInitialesRepository.chargerLesFormationsInitialesDansLeCMS(formationsInitialesASauvegarder, flux.nom);
+		const formationsInitialesEnErreur = formationInitialesASupprimerEnErreur.concat(formationsInitialesASauvegarderEnErreur);
+
+		await this.enregistrerLesResultatsDuChargementDansLeMinio(
```

Comment:
```suggestion
		await this.enregistrerLesResultats(
```

(on dépend d'un repository, potentiellement le repository injecté pourra stocker ailleurs que dans minio)

Reply:
On se perdait entre enregistrement et chargement du coup on a renommé comme ca pour comprendre.
C'est vrai qu'il faudrait pas préciser Minio mais je sais pas comment on pourrait faire 🤔

Reply:
Mmh, propositions : 
`enregistrerHistoriqueResultats` pour ce qui va dans le MinIO
et garder un `chargerLesFormationsInitiales` pour ce qui va dans le CMS ?